### ISSUE TITLE
Too much FOV and mis-aligned 3d elements

### ISSUE THREAD

**Issue by Michael-Z-Freeman** [SuiMachine/khg-d3d10drv#1]
Hi. Great to see this renderer for this interesting game.

However I get an enormous FOV on the menu 3D scene. Also when I select my character the 3D representation on the right is misaligned. Also there is a mouse cursor with hour glass overlayed over the screen all the time.

I followed the instructions about installation. 

The game already works OK with the updated renderer as recommended in this guide - https://fdossena.com/?p=khg/index.frag (which only goes up to 102x768).

Cheers.

**Comment by SuiMachine** [comment]
I never liked the dgVoodoo 2 solution, as it felt a bit sluggish to control as well as it hasn't worked very well in windowed mode. My hope was to get someone like Silent involved in the project to squish out few remaining bugs related to disposing the renderer, but I am just terrible at convincing people.

As for FOV, it's a bit of an oversight, as I was writing proper FOV scaling for widescreen monitors and just never finished it and then totally forgot about it, since it's only used when AutoFOV is true. Will just change it to false by default for now.

**Comment by Michael-Z-Freeman** [comment]
Was this fix in the updated download ? I tried it but it still has the same problem. I tried starting a new game as all the saved games had disappeared (come back once I restore original renderer) and then get ...

---------------------------
Critical Error
---------------------------
General protection fault!

History: UObject::GetFullName <- UObject::Destroy <- UObject::GetFullName <- UObject::ConditionalDestroy <- UObject::GetFullName <- UWindowsViewport::TryRenderDevice <- UWindowsViewport::Exec <- APlayerPawn::execConsoleCommand <- UObject::ProcessEvent <- (Console Transient.Console0, Function Engine.Console.Menuing.KeyEvent) <- UEngine::InputEvent <- UWindowsViewport::CauseInputEvent <- WM_KEYDOWN <- UWindowsViewport::WndProc <- WWindow::StaticProc <- DispatchMessage <- 00020632 256 <- MessagePump <- MainLoop
---------------------------
OK   
---------------------------

**Comment by SuiMachine** [comment]
Yeah, that is is one error I've been trying forever to fix (happening on closing the game or when cinematics should start playing, so most likely when disposing the D3D device), which sadly happens in Core.dll's function and not inside my renderer's functions, making it really hard to debug without source code for the  game's Core.dll library.
